src/styles/global.css

```diff
@@ -27,4 +27,10 @@ body {
 
 a:hover {
   text-decoration: underline;
+}
+
+body:has(a[data-twitch]:hover,a[data-twitch]:focus) img {
+  @media (prefers-reduced-motion: no-preference) {
```

Comment:
[prefers-reduced-motion: Taking a no-motion-first approach to animations](https://www.tatianamac.com/posts/prefers-reduced-motion) from Tatiana Mac on prefers-reduced-motion is a great read.